src/index.ts

```diff
@@ -0,0 +1,84 @@
+export interface Location {
+  name?: string;
+  city?: string;
+  zip?: string;
+  state?: string;
+  address?: string;
+  addressLine?: string;
+  address2?: string;
+  vagueAddress?: string;
+  coordinates?: {latitude: string; longitude: string; lat?: string; lng?: string};
+  coords?: {latitude: string; longitude: string; lat: string; lng: string};
```

Comment:
If we're aiming to match `coords` with `coordinates`, shouldn't `lat` and `lnt` be optional too?
```suggestion
  coords?: {latitude: string; longitude: string; lat?: string; lng?: string};
```

Reply:
Btw, it's so weird to see 3 places where we have coordinates, and more weird than that seeing that we have `latitude`, `longitude`, `lat` and `lng` instead of just one consistent pattern, but i believe some apps use both of them, but this just make this lib more complicated to be maintaned...